sys/sys/elf_common.h

```diff
@@ -1056,6 +1056,13 @@ enum MipsCheriFlags {
 #define	R_AARCH64_TLSDESC 	1031	/* Identify the TLS descriptor */
 #define	R_AARCH64_IRELATIVE	1032
 
+#define	R_MORELLO_CAPINIT	0xe800
+#define	R_MORELLO_CAPINIT	0xe800
+#define	R_MORELLO_GLOB_DAT	0xe801
+#define	R_MORELLO_JUMP_SLOT	0xe802
+#define	R_MORELLO_RELATIVE	0xe803
+#define	R_MORELLO_IRELATIVE	0xe804
```

Comment:
ISTR the LLVM toolchain expresses them in decimal in its files, and so do the other relocations here.